components/Tabs/Tabs.js

```diff
@@ -1,58 +1,94 @@
+                                               //                         0 = 'tab', 1 = 'active-tab'
 
     // Select all of the elements with the .card class on them
-    const cards = ;
+    const cards = document.querySelectorAll('.card'); //  = NodeList(23)
+                                                            //    0 = div.card, 1 = div.card, ...
+
     // Iterate through the NodeList setting the display style each one to 'none'
-    cards.forEach()
-    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class, nothing to update here
-    this.cards.forEach(card => card.selectCard());
-  }
+    // This clears cards and then rebuilds view based on tab selected
+
+    cards.forEach( item => item.style = 'display: none' ); // cards  = NodeList(23)
+                                                           // 0 = div.card, 1 = div.card
+    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class,
+    // nothing to update here
+              // e.g. this = TabLink{element: ,tabData: "jquery", cards}
+              // this.cards  =   an array of TabCard objects
+    this.cards.forEach(card => card.selectCard());  // e.g. this = TabLink{element:, tab-data='all', cards}
+  }                                                 // this.cards  = array of TabCard instances
 }
 
 class TabCard {
   constructor(element){
     // Assign this.element to the passed in element.
-    this.element;
+                              // this = TabCard(element)
+    this.element = element;   //  = div.card{align: "", title: "", lang: "", ...}
   }
   selectCard(){
     // Update the style of this.element to display = null
-    this.element;
+            // this = TabCard{element}
+            // this.element = div.card{align: "", title: "", ...}
+            // this.element.style =  CSSStyleDeclaration{0: "display", ....
+                   //  display: none;
+    this.element.style = 'display: null';  // each card added one at a time positioned with flex
+                  //    display: ""
   }
 
 }
 
 // Create a reference to all ".tab" classes
-let tabs = document.querySelectorAll();
-// Map over the array and convert each tab reference into a new TabLink object.  Pass in the tab object to the Tabs class.
-tabs = Array.from(tabs).map()
-
-//Once you are complete, call the .select method on the first tab
\ No newline at end of file
+let tabs = document.querySelectorAll('.tab');  // = NodeList(9)
+                                               // 0 = div.tab.active-tab, 1 = div.tab, 2 = div.tab, ...
+// Map over the array and convert each tab reference into a new TabLink object.
+// Pass in the tab object to the Tabs class.
+tabs = Array.from(tabs).map(item => new TabLink(item)); // = Array(9)  => TabLink[0]
+           // 0 = TabLink{element: , tabData = "all", cards:}
+           // 1 = TabLink{element: , tabData = "javascript", cards:}
```

Comment:
I can tell you have a pretty great understanding of these components by these very thorough notes you've created.